bin/pytorch_inference/CCommandParser.h

```diff
@@ -50,21 +50,27 @@ class CCommandParser {
 public:
     static const std::string REQUEST_ID;
     static const std::string TOKENS;
-    static const std::string INPUTS;
     static const std::string VAR_ARG_PREFIX;
     static const std::string UNKNOWN_ID;
 
     using TUint64Vec = std::vector<std::uint64_t>;
     using TUint64VecVec = std::vector<TUint64Vec>;
     using TDoubleVec = std::vector<double>;
 
+    //! The incoming JSON requests contain a 2D array of tokens representing
+    //! a batch of inference calls. To avoid copying, the input tensor
+    //! should be created directly from contiguous data so the 2D token
+    //! array is read into a 1D vector of size w * h where w & h are the
+    //! dimensions of in the JSON input. The secondary arguments are
+    //! treated in the same manner.
     struct SRequest {
+        std::int64_t w;
+        std::int64_t h;
```

Comment:
nit: these names seem a bit brief to me and also violate the coding standards. How about `s_NumberInferences` and `s_FeatureVectorDimension`. I think it would also make what the code is doing clearer.

Reply:
I changed these to `s_NumberInferences` and `s_NumberInputTokens`